### PR TITLE
Added Interface for telemetry and implemented using Azure Applicaiton…

### PR DESCRIPTION
**Pull request by ashirs-ms-office**
… insights

### CODE REVIEW

File: Microsoft.Web.Portal.Common/Telemetry/ApplicaitonInsightsTelemetry.cs [hunk 1]
```diff
@@ -0,0 +1,95 @@
+﻿using System;
```
Comment (kevnickel) [inline Microsoft.Web.Portal.Common/Telemetry/ApplicaitonInsightsTelemetry.cs]: Fix spelling on the filename

File: Microsoft.Web.Portal.Common/Telemetry/ApplicaitonInsightsTelemetry.cs [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Diagnostics.Contracts;
+using Microsoft.ApplicationInsights;
+namespace Microsoft.Web.Portal.Common.Telemetry
+{
+    public class ApplicaitonInsightsTelemetry : ITelemetry
```
Comment (kevnickel) [inline Microsoft.Web.Portal.Common/Telemetry/ApplicaitonInsightsTelemetry.cs]: Fix spelling on the classname

File: Microsoft.Web.Portal.Common/Telemetry/ApplicaitonInsightsTelemetry.cs [hunk 3]
```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
```
Comment (kevnickel) [inline Microsoft.Web.Portal.Common/Telemetry/ApplicaitonInsightsTelemetry.cs]: Remove and sort

**Review by kevnickel — CHANGES_REQUESTED** [review]

File: Microsoft.Web.Portal.Common/Telemetry/ApplicaitonInsightsTelemetry.cs (hunk 2 above)
Comment (ashirs-ms-office) [reply Microsoft.Web.Portal.Common/Telemetry/ApplicaitonInsightsTelemetry.cs]: Done

File: Microsoft.Web.Portal.Common/Telemetry/ApplicaitonInsightsTelemetry.cs (hunk 3 above)
Comment (ashirs-ms-office) [reply Microsoft.Web.Portal.Common/Telemetry/ApplicaitonInsightsTelemetry.cs]: done

**Review by kevnickel — APPROVED** [review]
Looks good!

**Pull request merged**